docs/Klasse_Datasett.adoc

```diff
@@ -592,6 +645,19 @@ dqvno:isAuthoritative
 |Kravsnivå| Valgfri
 |===
 
+=== Datasett: siste (dcat:last) [[Datasett-siste]]
```

Comment:
Tilsvarende som for dcat:first nevnt ovenfor, dvs. flyttes til klassen Datasett serie.

Reply:
Fiksa i 92be7818bdee90bc2ae5f9423865eb0e45ed83b4